zebra-state/src/request.rs

```diff
@@ -731,6 +731,14 @@ pub enum ReadRequest {
     ///
     /// Returns a type with found utxos and transaction information.
     UtxosByAddresses(HashSet<transparent::Address>),
+
+    /// Looks up a block hash by height in the current best chain.
+    ///
+    /// Returns
+    ///
+    /// * [`ReadResponse::BlockHash(Some(hash))`](ReadResponse::BlockHash) if the block is in the best chain;
+    /// * [`ReadResponse::BlockHash(None)`](ReadResponse::BlockHash) otherwise.
+    BestChainBlockHash(Height),
```

Comment:
Do you think we should put this new state request behind the getblocktemplate-rpcs feature?